src/main/java/core/basesyntax/figures/FigureData.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.figures;
+
+public interface FigureData {
```

Comment:
Let`s make 2 interfaces (AreaCalculator and Drawable for example) with methods getArea() and draw() so we can use these Interfaces in other classes later. For e[ample we can use a Drawable interface for Graph class, but graphs do not have an area to calculate